client/app/src/main/java/io/mosip/registration/app/activites/JobServiceActivity.java

```diff
@@ -81,36 +80,33 @@ public CustomListViewAdapter(@NonNull Context context, int resource, @NonNull Li
         @Override
         public View getView(int position, @Nullable View convertView, @NonNull ViewGroup parent) {
 
-            JobServiceModel jobServiceModel = this.mObjects.get(position);
             ViewHolder mainViewHolder = null;
             if (convertView == null) {
                 LayoutInflater layoutInflater = LayoutInflater.from(getContext());
                 convertView = layoutInflater.inflate(layout, parent, false);
 
                 ViewHolder viewHolder = new ViewHolder();
                 viewHolder.jobName = convertView.findViewById(R.id.jobName);
+                viewHolder.jobLastSyncTime = convertView.findViewById(R.id.last_sync_time_text);
+                viewHolder.jobNextSyncTime = convertView.findViewById(R.id.next_sync_time_text);
                 viewHolder.triggerJobButton = convertView.findViewById(R.id.triggerJob);
                 viewHolder.toggleActiveButton = convertView.findViewById(R.id.toggleActive);
-                viewHolder.toggleActiveButton.setChecked(jobServiceModel.getEnabled());
-
-                if (jobServiceModel.getActive() && jobServiceModel.getImplemented())
-                    viewHolder.jobName.setText(jobServiceModel.getName());
-                else {
-                    String name = "";
-                    if (!jobServiceModel.getActive())
-                        name = jobServiceModel.getName() + " (Not Active)";
-                    else
-                        name = jobServiceModel.getName() + " (Not implemented)";
-                    viewHolder.jobName.setText(name);
-                    viewHolder.toggleActiveButton.setEnabled(false);
-                    viewHolder.triggerJobButton.setEnabled(false);
-                }
 
                 convertView.setTag(viewHolder);
             }
+            JobServiceModel jobServiceModel = this.mObjects.get(position);
 
             mainViewHolder = (ViewHolder) convertView.getTag();
 
+            mainViewHolder.jobName.setText(jobServiceModel.getName());
+            mainViewHolder.jobLastSyncTime.setText("Last Sync Time:" + jobServiceModel.getLastSyncTime());
```

Comment:
Add these constants in the strings.xml under values to support internationalization.

Reply:
done